test/features/icons/icons.test.js

```diff
@@ -45,27 +44,124 @@ describe('Icon Suppprt', () => {
 
   it('Fetches successfully with cache control enabled', async () => {
     const otherIcons = [createTag('span', { class: 'icon icon-play' })];
+    document.body.appendChild(otherIcons[0]);
+
     await loadIcons(otherIcons, config);
+    const svg = otherIcons[0].querySelector('svg');
+    expect(svg).to.exist;
```

Comment:
nit, you can save 1 line:
```suggestion
    expect(otherIcons[0].querySelector('svg')).to.exist;
```